src/commons.ts

```diff
@@ -97,14 +96,12 @@ export default class Commons {
     }
 
     let uploadStopped: boolean = true;
-    Commons.extensionWatcher = chokidar.watch(this.en.ExtensionFolder, {
-      depth: 0,
-      ignoreInitial: true
-    });
-    Commons.configWatcher = chokidar.watch(this.en.PATH + "/User/", {
-      depth: 2,
-      ignoreInitial: true

```

Comment:
It would be great if you can write details how are you handling these situations on `FileSystemWatcher`

Reply:
The watcher doesn't watch the `add` event now, so I think it's not problem to ignore this option.

Reply:
Now, I checked this behavior.
FileSystemWatcher and chokidar with `ignoreInitial: true` are same. 😄 

Add the following code for check, and `Added: ` logs are not appeared when the window reloads.

```ts
    Commons.configWatcher.onDidCreate(async (uri: vscode.Uri) => {
      const path: string = uri.path;
      console.log("Added: " + path);
    });
```

I also checked the case of `ignoreInitial: false`. https://github.com/nekonenene/code-settings-sync/commit/59412864cff08cc078d2f4aec7584a10c0e3eb5d

<img width="207" alt="sc 619" src="https://user-images.githubusercontent.com/11713748/52523853-565d6f00-2cd9-11e9-99eb-c375aef6988f.png">